### PR TITLE
(#250) breadcrumbs and storybook fix

### LINKED ISSUE #250: Design: Mobile Navigation Changes
> **ESTIMATE** Story Points TBD

> Feature/Main User Story: Mobile Nav

### Description of Design request:

Based on input from our navigation testing, work on redesigning our mobile nav to be able to "refresh" the mobile nav as users click in it to traverse our hierarchy (instead of closing the menu and refreshing the page, forcing users to reopen the menu), but also provide a way for the user to get to the page.

Based on conversations, Lynn/Fedora are ok with deviating from USWDS on mobile nav, since our navigation is much deeper and needs more custom work for it to be usable.  A secondary note is that whatever design we come up with will have to work for DOCs as well.

##

### List of sub-tasks:
*Add links to related tickets below*
- [ ] Sub-Task
- [ ] Sub-Task
- [ ] Sub-Task

##

### Design Functionality:
*Describe how the design should function, e.g. color changes on hover*
1. Specification 
2. Specification  
3. Specification 

##

### Design specs / InVision:
*Design specs indicated in the InVision links below. Please view in the inspect mode along with the comments panel*
1. https://publicis.invisionapp.com/share/4G11TDSW58SE

##

### Assets:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

##

### Design Research & Inspiration:
*Space to add inspiration, moodboards,etc*

##

### LINKED ISSUE #299: Storybook Testing is unable to use Images
Our Storybook configuration cannot use images in CSS. Additionally, it seems that we do not add any vendor prefixing to the css generated in our stories. (Which is really evident when masking images)

### PR DESCRIPTION
**Pull request by bryanpizzillo**
This is viewable at https://designsystem-dev.cancer.gov/pr-301

Closes #299 
Closes #250

### CODE REVIEW

**Comment by mworrest** [comment]
I am getting a page not found error.

**Comment by karlikpj** [comment]
https://designsystem-dev.cancer.gov/pr-301/components/breadcrumb

you should be viewing this link - with the PR number in it?

**Comment by mworrest** [comment]
That's better--thanks. Our breadcrumbs are set to size 14px in the comps. Could you please update?

File: packages/ncids-css/scss/components/usa-breadcrumb/internal/_shared.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        "after",
+        0,
+        baseline,
+        $theme-breadcrumb-background-color
+      );
+    }
+
+    // Style overrides for separator using non-token values
+    &:not(:last-child)::after {
+      // Magic number to center icon
```
Comment (arcepaul) [inline packages/ncids-css/scss/components/usa-breadcrumb/internal/_shared.scss:119]: how is this centering the icon?

File: testing/ncids-css-testing/postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: [
```
Comment (arcepaul) [inline testing/ncids-css-testing/postcss.config.js:2]: tabbing is effed up

File: packages/ncids-css/scss/components/usa-breadcrumb/internal/_shared.scss (hunk 1 above)
Comment (karlikpj) [reply packages/ncids-css/scss/components/usa-breadcrumb/internal/_shared.scss:119]: unsure - default from uswds

File: testing/ncids-css-testing/postcss.config.js (hunk 2 above)
Comment (karlikpj) [reply testing/ncids-css-testing/postcss.config.js:2]: updating

**Review by mworrest — APPROVED** [review]
Looks great from design standpoint!

**Pull request merged**